### DISCUSSION TITLE
XP12 - stutters while creating autoo

### DISCUSSION THREAD

**Discussion by GEC42P** [kubilus1/autoortho#292] [General]
Hello again,

1. flight from EFHK to ENBR, 
- autoo generates first around 20 mins
- departure ok
- Alt 26000 3 mins XP12 freezes, autoo works
- Alt 27000 2 mins XP12 freezes, autoo works
- I did nothing so systems revovers without breaking
Information : 
Disk with Cache 2% load, Memory 15 GB load, CPU 15 % load
Vatsim connected, navigraph connected
Network load 500 kbit/s (Bandwith 500Mbit)

What is the reason for this ?

/Manfred

**Comment by dpdurst** [comment]
You are flying a jet faster than AutoOrtho can stream the data from the servers. This is not an AO issue.

**Reply by GEC42P** [reply]
yes I agree, i tested it now and deleted autoo because this way it's not usable.
basically it is a great idea and a good implemented tool but has his limitations.
Bad is that this is not described anywhere and you as the user has to find out.
But it is freeware .... so no complains.

Manfred

**Comment by GEC42P** [comment]
Wouldn't it make sense to implement a function to download everything for a given area like EU into the cache and only check whether a file is newer then the existing one ?
What I see is that Autoo uses less diskspace than the Ortho4XP so 100 GB Cache .... it doesn't matter

Manfred

**Reply by d41k4n** [reply]
Duplicate of https://github.com/kubilus1/autoortho/discussions/295 and answered with https://github.com/kubilus1/autoortho/discussions/295#discussioncomment-6733552
